src/web/forms/fields/AutoCalc/functions.js

```diff
@@ -26,4 +26,17 @@ export default {
       return null;
     }
   },
+
+  'asha:sum': (a, b, c, d, e, f) => {
```

Comment:
It's better to accept arbitrary number of arguments.

(ref: https://davidwalsh.name/spread-operator )

Reply:
Done in commit 78a063b6f29d05a465e3256e4f9044f058fe989e

Reply:
🆒